modules/op-results-search-records.adoc

```diff
@@ -0,0 +1,73 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="results-search-records_{context}"]
+= Searching for records
+
+[role="_abstract"]
+You can search for records using Common Expression Language (CEL) queries. As each record contains full YAML information for a pipeline run or task run, you can find records by many different criteria.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility and prepared its environment to quety {tekton-results}.
+
+.Procedure
+
+* To search for records using a CEL query, run the following command:
++
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} --filter=<CEL query> <namespace-name>/result/-
+----
++
+Replace `<namespace-name>` with the namespace in which the pipeline runs or task runs were created. Alretnatively, to search the records within a single result, use the following command:
++
+[source, terminal]
+----
+$ opc results records list --addr ${RESULTS_API} --filter=<CEL query> <result-name>
+----
++
+Replace `<result-name>` with the full name of the result.
+
+
+.Example CEL queries for records
+|===
+| Purpose | CEL query
+
+| Records of all task runs or pipeline runs that failed
+| `!(data.status.conditions[0].status == 'True')`
+
+| Records where the name of the `TaskRun` or `PipelineRun` custom resource (CR) was `run1`
+| `data.metadata.name == 'run1'`
+
+| Records for all task runs that were started by the `PipelineRun` CR named `run1`
+| `data.metadata.labels['tekton.dev/pipelineRun'] == 'run1'`
```

Comment:
```suggestion
| `data_type == 'TASK_RUN' && data.metadata.labels['tekton.dev/pipelineRun'] == 'run1'`
```
The filter should work, but the description says TaskRun only so you can add this for finer control.